crates/sequencer/src/runner.rs

```diff
@@ -607,18 +603,25 @@ where
             self.config.max_l2_blocks_per_commitment,
         );
 
-        tokio::spawn(commitment_service.run(
-            self.storage_manager.clone(),
-            self.l2_block_hash,
-            shutdown_signal.clone(),
-        ));
+        tokio::spawn(
+            commitment_service
+                .run(
+                    self.storage_manager.clone(),
+                    self.l2_block_hash,
+                    shutdown_signal.clone(),
+                )
+                .instrument(span!(Level::INFO, "CommitmentService")),
+        );
 
-        tokio::spawn(da_block_monitor(
-            self.da_service.clone(),
-            da_height_update_tx,
-            self.config.da_update_interval_ms,
-            shutdown_signal.clone(),
-        ));
+        tokio::spawn(
+            da_block_monitor(
+                self.da_service.clone(),
+                da_height_update_tx,
+                self.config.da_update_interval_ms,
+                shutdown_signal.clone(),
+            )
+            .instrument(span!(Level::INFO, "L1BlockMonitor")),
```

Comment:
Why not put name = L1BlockMonitor to `#[instrument(name=...)] fn da_block_monitor`?